logfire/_internal/metrics.py

```diff
@@ -353,7 +361,7 @@ def _create_real_instrument(self, meter: Meter) -> UpDownCounter:
         return meter.create_up_down_counter(self._name, self._unit, self._description)
 
 
-if Gauge is not None:  # pragma: no branch
+if Gauge is not None:  # pragma: no cover
```

Comment:
Surely this is covered?

Reply:
My bad, I was just trying to readd what was previously there.